app/src/app/api/v0/organizations/route.ts

```diff
@@ -0,0 +1,22 @@
+import { Organization } from "@/models/Organization";
+import { randomUUID } from "node:crypto";
+import { createOrganizationRequest } from "@/util/validation";
+import { NextResponse } from "next/server";
+import UserService from "@/backend/UserService";
+import { apiHandler } from "@/util/api";
+
+export const POST = apiHandler(async (req, { params, session }) => {
+  UserService.validateIsOefAdmin(session);
```

Comment:
Mentioned this on the other PR, but let's use the existing `Admin` role for global admin functionality (as it is already being used in the API). We can then introduce another field on e.g. the OrganizationUser relationship table to define the role a user has inside of an organization, or add another user role if necessary for organization- / project-level 'administrators' (which could be called `owners` or `managers` or the organization perhaps).